docs/source/projections/eqc.rst

```diff
@@ -4,7 +4,95 @@
 ********************************************************************************
 
+The simplist of all projections. Standard parallels (0° when omitted) may be specified that determine latitude of true scale (k=h=1).
+
++---------------------+----------------------------------------------------------+
+| **Classification**  | Conformal cylindrical                                    |
++---------------------+----------------------------------------------------------+
+| **Available forms** | Forward and inverse                                      |
++---------------------+----------------------------------------------------------+
+| **Defined area**    | Global, but best used near the equator                   |
++---------------------+----------------------------------------------------------+
+| **Implemented by**  | Marinus of Tyre                                          |
++---------------------+----------------------------------------------------------+
+| **Options**                                                                    |
++---------------------+----------------------------------------------------------+
+| `+lat_ts`           | Latitude of true scale. Defaults to 0.0                  |
++---------------------+----------------------------------------------------------+
+| `+lat_0`            | Center of the map                                        |
++---------------------+----------------------------------------------------------+
+
 .. image:: ./images/eqc.png
    :scale: 50%
    :alt:   Equidistant Cylindrical (Plate Caree)  
 
+Usage
+########
+
+Because of the distortions introduced by this projection, it has little use in navigation or cadastral mapping and finds its main use in thematic mapping. In particular, the plate carrée has become a standard for global raster datasets, such as Celestia and NASA World Wind, because of the particularly simple relationship between the position of an image pixel on the map and its corresponding geographic location on Earth.
+
+The following table gives special cases of the cylindrical equidistant projection. 
+
++---------------------------------------------------------+--------------------------+
+| Projection Name                                         | (lat ts=) :math:`\phi_0` |
++---------------------------------------------------------+--------------------------+
+| Plain/Plane Chart                                       | 0◦                       |
++---------------------------------------------------------+--------------------------+
+| Simple Cylindrical                                      | 0◦                       |
++---------------------------------------------------------+--------------------------+
+| Plate Carrée                                            | 0◦                       |
++---------------------------------------------------------+--------------------------+
+| Ronald Miller—minimum overall scale distortion          | 37°30'                   |
```

Comment:
The "apostrophe" should be a "prime", cf. ' vs. ′ (though I can't really see a difference with my default font here, so it's rather minor, and probably not a big deal.)